src/stories/Blocks/form/form.scss

```diff
@@ -5,55 +5,57 @@
   padding-left: $s-md;
   padding-right: $s-md;
   box-sizing: border-box;
+}
 
-  &__header {
-    margin: $s-2xl 0 $s-2xl $s-md;
+.dpl-form__header {
```

Comment:
Mmmmm, 🤔 why?

Reply:
https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/develop/docs/scss.md#bem-naming-convention

>(&__ should be avoided, to avoid massive indention.)

This is something we have started to do, moving forward.
Another value, apart from avoiding indention, is also that when you use `&__` you make it really hard to debug things.

Let's say you have a class called `dpl-form__header` that you can see in the browser, and you want to find it in the site files - if you just search for "dpl-form__header" it wont show up.